app/_how-tos/secure-api-with-tls.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Secure your API with TLS using SNIs
+content_type: how_to
+related_resources:
+  - text: SNI entity
+    url: /gateway/entities/snis/
+  - text: Certificates
+    url: /gateway/entities/certificate/
+
+
+products:
+    - gateway
+tier: enterprise
+
+works_on:
+    - on-prem
+
+entities: 
+  - certificate
+  - sni
+  - route
+  - service
+
+tldr:
+    q: How do I set up {{site.base_gateway}} with TLS?
+    a: By creating an SSL [Certificate](/gateway/entities/certificate), and associating it with an [SNI](/gateway/entities/sni), you can force {{site.base_gateway}} to handle encrypted traffic.
+tools:
+    - deck
+
+prereqs:
+  inline:
+    - title: Certificate
+      include_content: prereqs/certificate
+      icon_url: /assets/icons/file.svg
+    - title: Configure environment variables
+      content: |
+        Set the following variables: 
+        * `CERT_ID`: The UUID set when you associated a Certificate with {{site.base_gateway}}.
+        For example: 
+        ```sh
+        export CERT_ID=3f19b7d1-705f-422a-a116-7dc8282efe21
+        ```
+        icon_url: /assets/icons/file.svg
```

Comment:
```suggestion
```
If you're going to leave the CERT_ID step in the certificates prereq, remove this. If you're doing it the opposite way, ignore this suggestion.